Jotify/SceneDelegate.swift

```diff
@@ -35,8 +54,23 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate, UNUserNotificationCente
             }
         }
 
-        // pull up recent note widget launched app
-        maybePressedRecentNoteWidget(urlContexts: connectionOptions.urlContexts)
+        if let urlContext = connectionOptions.urlContexts.first {
+            Airbridge.trackDeeplink(url: urlContext.url)
+            
+            let isHandled = Airbridge.handleDeeplink(url: urlContext.url) { url in
+                self.handleDeepLink(url)
+            }
+            
+            if !isHandled {
+                maybePressedRecentNoteWidget(urlContexts: connectionOptions.urlContexts)
+            }
+        }
+        
+        Airbridge.handleDeferredDeeplink { url in
+            if let url = url {
+                self.handleDeepLink(url)
+            }
+        }
```

Comment:
Implemented comprehensive deep link handling for scene(_:willConnectTo:options:). This includes tracking deep links, handling deferred deep links, and maintaining the widget functionality. The implementation follows Airbridge's recommended pattern for deep link handling in UIKit apps with SceneDelegate.